xml/public-cloud-setup.xml

```diff
@@ -110,14 +110,26 @@
       <para> Set the hostname to a name of your choice. Please note
        it is important to provide the <replaceable>system name</replaceable>
        and not the <replaceable>fully qualified host name</replaceable>
-       to the command. In the following example we use
-       <emphasis role="strong">suma</emphasis> as hostname and the
-       domain name is set to <emphasis role="strong">cloud.net</emphasis>.
-       </para>
-      <screen>$ hostnamectl set-hostname suma
-$ hostname -f
+       to the hostnamectl command.
+
+       On the assumption that the fully qualified host name of the
```

Comment:
We should say:

It is expected that the Fully Qualified Domain Name (FQDN) is set by the cloud framework, in this example we use cloud_instance.cloud.net